test/unit/helpers/tokens.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from "chai";
+
+import { createDaoUserEntity } from "../mocks/daoUserEntityFactory";
+import { computeBalances, bigIntToNum } from "../../../src/helpers/tokens";
+import { createOfferEntity } from "../mocks/offerEntityFactory";
+import { add, getUnixTime, sub } from "date-fns";
+
+describe("Tokens helpers", () => {
+  it("should correctly transform a bigInt to an integer", () => {
+    expect(bigIntToNum(BigInt("42000000000000000000"))).to.eql(42);
```

Comment:
you can use `parseEther("42")` I guess.

Reply:
the problem is `parseEther` returns a `BigNumber`, whereas the graph returns a `BigInt`, so I need to transform that (via `formatEther`)